onnxruntime-sys/build.rs

```diff
@@ -484,11 +508,18 @@ fn prebuilt_archive_url() -> (PathBuf, String) {
         ORT_VERSION,
         TRIPLET.os.archive_extension()
     );
-    let prebuilt_url = format!(
-        "{}/v{}/{}",
-        ORT_RELEASE_BASE_URL, ORT_VERSION, prebuilt_archive
-    );
-
+    let prebuilt_url= match TRIPLET.os {
+        Os::Android =>format!(
```

Comment:
めちゃくちゃ細かいですが:

```suggestion
        Os::Android => format!(
```

(この1行下が100文字を超えているため、Rustfmtが仕事を放棄しています)

Reply:
URLのほうを`ORT_RELEASE_BASE_URL`と同じように別の箇所で`ORT_MAVEN_RELEASE_BASE_URL`みたいな感じで定義しちゃっても良いかもですね！

Reply:
確かにそうですね、ORT_MAVEN_RELEASE_BASE_URL  として切り出します